### PR TITLE
Check if tty is available when getting columns + add other ways to obtain columns

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**
Fixes https://github.com/ZOSOpenTools/meta/issues/468

### CODE REVIEW

File: include/common.sh [hunk 1]
```diff
@@ -316,8 +316,14 @@ ansiline()
 
 getScreenCols()
 {
-  # Note tput does not handle ssh sessions too well...
-  stty | awk -F'[/=;]' '/columns/ { print $4}' | tr -d " "
+  if  [ -t 1 ] && [ -t 2 ]; then
```
Comment (MikeFultonDev) [inline include/common.sh:319]: I had to look this up - might just be me - but would it make sense to have a comment that says that -t fd tests if the descriptor is opened on a terminal?

Comment (IgorTodorovskiIBM) [reply include/common.sh:320]: thanks, added a comment

**Review by MikeFultonDev — APPROVED** [review]
LGTM

**Pull request merged**